Description: Drop references to Sphinx extensions not provided in Debian
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2021-05-10
Forwarded: not-needed

--- a/doc/conf.py
+++ b/doc/conf.py
@@ -85,9 +85,9 @@
     "nbsphinx",
 #    "sphinx_autosummary_accessors",
     "sphinx.ext.linkcode",
-    "sphinxext.opengraph",
-    "sphinx_copybutton",
-    "sphinxext.rediraffe",
+#    "sphinxext.opengraph",
+#    "sphinx_copybutton",
+#   "sphinxext.rediraffe",
     "sphinx_design",
     "sphinx_inline_tabs",
 ]
@@ -222,7 +222,7 @@
 # -- Options for HTML output ----------------------------------------------
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_book_theme"
+#html_theme = "sphinx_book_theme"
 html_title = ""
 
 html_context = {
